picture-stealer/src/main/java/com/bobocode/picturestealer/entity/CameraEntity.java

```diff
@@ -0,0 +1,60 @@
+package com.bobocode.picturestealer.entity;
+
+import jakarta.persistence.Cacheable;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.persistence.Version;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+import org.hibernate.annotations.CreationTimestamp;
+
+import java.time.OffsetDateTime;
+import java.util.UUID;
+
+/**
+ * @author "Maksym Oliinyk"
+ */
+@Getter
+@NoArgsConstructor
+@EqualsAndHashCode
+@ToString(onlyExplicitlyIncluded = true)
+@Cacheable
+@Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
+@Entity
+@Table(name = "camera")
+public class CameraEntity {
+
+    @Id
+    private String id;
+
+    @Column(name = "nasa_id", unique = true)
+    private String nasaId;
+
+    @Column
+    private String name;
+    @Column(name = "full_name")
+    private String fullName;
+
+    @CreationTimestamp
+    @Column(name = "created_at")
+    private OffsetDateTime createdAt;
+
+    @Version
```

Comment:
@WHALE88 I don't get why we need an optimistic locking for this task.

Reply:
@tboychuk In fact, it is needed for competing requests for the same sol. There was an idea to use the retry mechanism and intercept the optimistic lock. It is not really needed here, since the database has a unique constraint on nasa_id, and in the case of competing requests, one will still fail with an error.  Another thing, I wanted to mock this app a little in the future and to permit the update of these entities

Reply:
Currently deleted